rasterio/__init__.py

```diff
@@ -23,11 +23,11 @@
                     os.add_dll_directory(p)
 
 
-from rasterio._show_versions import show_versions
+from rasterio._show_versions import show_versions  # noqa: F401
```

Comment:
@snowman2 what would you think about putting `noqa: F401` etc in a config file instead of the code?

Reply:
I usually lean towards doing a subset. However, in this scenario I think a full file ignore makes sense. I also did this for `path.py`.